app/views/users/show.html.erb

```diff
@@ -4,7 +4,7 @@
   </div>
 </div>
 
-<% if current_user == @user || !@user.private? || current_user.leaders.include?(@user) %>
+<% if policy(@user).show_photos? %>
```

Comment:
Good job creating a policy to hide private users' feeds!

Reply:
+1